### ISSUE TITLE
Arguments/el passed to bound function?

### ISSUE THREAD

**Issue by andriijas** [mikeric/rivets#291]
Hi

im trying to do:

```
rv-show="view.validateField < model:name" data-field="#name"
```

I need to supply view.validateField with an argument because i reuse it for other elements, but im not sure how to do it? Not even the el is passed to validateField so i cant read the data arg.

Thanks

**Comment by Leeds-eBooks** [comment]
:+1:

**Comment by rquantz** [comment]
Yes, this seems like a pretty important feature. Also for dependent models, it would be good to be able to pass the model to its parent so you don't have to have circular references.

    <div rv-each-item="list.items">
      <li>
        { item.name }
        <a rv-on-click="list.destroyItem(item)">destroy</a>
      </li>
    </div>
